libraries/AP_Notify/RGBLed.cpp

```diff
@@ -262,7 +262,7 @@ void RGBLed::update_colours(void)
             // fast flashing green if disarmed with GPS 3D lock and DGPS
             // slow flashing green if disarmed with GPS 3d lock (and no DGPS)
             // flashing blue if disarmed with no gps lock or gps pre_arm checks have failed
-            bool fast_green = AP_Notify::flags.gps_status >= AP_GPS::GPS_OK_FIX_3D_DGPS && AP_Notify::flags.pre_arm_gps_check;
+            bool fast_green = AP_Notify::flags.gps_status > AP_GPS::GPS_OK_FIX_3D_DGPS && AP_Notify::flags.pre_arm_gps_check;
```

Comment:
Since this is also the topic in #6002 I think we should probably discuss it there.

The short form, is that DGPS can be achieved with other correction sources outside of RTK (I can inject RTCM data to a U-Blox 7 and get DGPS with it). SBAS really is an augmentation source, and can be used for more then just aiding, so I think it's pretty valuable to know if you have SBAS lock or not.

I'm not opposed to more indicators but I don't really want to give up the SBAS one.